public/app/features/serviceaccounts/components/ServiceAccountProfileRow.tsx

```diff
@@ -91,7 +90,7 @@ export const ServiceAccountProfileRow = ({ label, value, inputType, disabled, on
             onCancel={onCancelClick}
             disabled={disabled}
           >
-            <Trans i18nKey="serviceaccounts.service-account-profile-row.edit">Edit</Trans>
+            Edit
```

Comment:
(You could wrap this in `t()` instead, I think that's what other ConfirmButtons have done)

Reply:
yeah i'm kinda intentionally leaving it for now so we can look into how we can fix this everywhere 😄 right now, any `<Trans>` tag inside a component which does `React.cloneElement` will break 😬 

you've prompted me to actually make an issue instead of leaving it in my head 😂